docs/atmosphere/frontend_guidelines.html

```diff
@@ -3,19 +3,46 @@
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Make it accessible</span>: Be mindful of providing the <a href="{{ site.baseurl }}/atmosphere/accessibility.html">Accessibility</a> affordances.
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Validate HTML</span>: Run your code through <a href="https://validator.w3.org/" target="_blank">W3C's HTML Validator</a>. The easiest way is to validate it by direct input. Paste all the email code in the text box, click Check, then review the errors. When working with often complicated email code full of nested tables, it's easy to forget a closing tag which can really mess up your layout in some email clients. Pass it through this tool to check for errors. Note: Disregard errors that relate to the ESP templating syntax.
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Review the Cross-Email Client Guidelines</span>: We have Cross-Email Client Guidelines that set you up for success. Refer to the <a href="{{ site.baseurl }}/atmosphere/email_client_support.html">Email Client Support chart</a> to identify the top clients we support.
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Compress HTML</span>: In an effort to get email size as small as possible, compress the HTML. Smaller emails mean they load faster and avoid being clipped by email clients when they surpass file size maximums (Gmail's max is 102KB). Luckily, image sizes aren't included in this- only the text, full URLs, and HTML is counted.
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Upload images wisely</span>: Ideally, upload images to the ESP that's housing the email. Alternatively, if you're using Mandrill or otherwise need a space to store images, you can upload them to S3. Ask Engineering for access or assistence. Use https for images to avoid any mixed content warnings.
+
+</p>
+
+<p class="m-t-2-xs">
+  <span class="bold">Test it</span>: Developing emails that render well across email clients is a tricky beast. Luckily, our templates have been thoroughly tested to ensure you're being set up for success. That being said, as emails are being tweaked there's always the chance that it could look amiss in a given browser on a certain OS. The goal with testing is to catch these before emails are sent out so that we can confidently send high quality, high performing, on-brand emails. Email on Acid is a handy tool for testing emails across email clients. Use the profile named "Treehouse Email Client Matrix" to test the email against the email clients defined in our <a href="{{ site.baseurl }}/atmosphere/email_client_support.html">Email Client Support chart</a>.
+</p>
+
+<h3 class="m-t-3-xs">Tips for ensuring consistent rendering across email clients</h3>
+<p class="m-t-2-xs">
+  <span class="bold">Insert tip here</span>: x
+</p>
```

Comment:
Hmm, I'm not sure what that means so I'd like to get your help. Do you have time to chat after the design systems meeting today?